app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,127 @@
+class TweetsController < ApplicationController
+
+
+	get '/tweets' do
+
+		if logged_in?
+			@tweets = Tweet.all
+
+			erb :'tweets/tweets'
+		else
+			redirect to '/login'
+		end
+	end
+
+
+	post '/tweets' do
+		if logged_in?
+			if params[:tweet] == ""
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.